src/main/scala/uk/gov/hmrc/ct/computations/CPQ17.scala

```diff
@@ -41,5 +41,6 @@ case class CPQ17(value: Option[Boolean]) extends CtBoxIdentifier(name = "Trading
 }
 
 object CPQ17 {
-  val lossReform2017 = LocalDate.parse("2017-04-01")
+  def lossReform2017Applies(to: StartDate): Boolean = to.value.isAfter(lossReform2017)
```

Comment:
That's wrong isn't it

It is meant to be EndDate on or after 1/4/2017.

Reply:
:+1: For defining the type. It made the problem to be visible.